index.js

```diff
@@ -24,7 +24,7 @@ var settings = {
   useEnv: true,
   usePkg: true,
   cwd: process.cwd(),
-  home: osHomedir(),
+  home: osHomedir,
```

Comment:
I'll might change this back to calling `os.homedir()` every time. It likely makes no difference for the vast majority of cases (config usually is processed once at app startup).